### ISSUE TITLE
Possibly download library dependencies into normal .haxelib?

### ISSUE THREAD

**Issue by Jarrio** [tobil4sk/haxe-git-submodule-manager#2]
When adding a library with this, if the library has a few dependencies you have to manually add them individually. Most of the time I don't actually need to version manage those as the parent project will typically handle that. Would it be possible to add a way to have gsm trigger haxelib's dependency check and just install them straight into the `.haxelib` folder?

**Comment by tobil4sk** [comment]
I think this should be possible, would just need gsm to be able to handle lib.haxe.org libraries. It would still have to keep track of the dependency versions by itself because you don't know whether the library locks down its dependencies (also, haxelibs locking down their dependencies is not scalable, due to how libraries work in Haxe).

I'd probably prefer to keep this behind a flag though, like `--with-dependencies` or something like that.

**Comment by Jarrio** [comment]
Yeah that's totally understandable, just a way to have it auto manage this would be nice. Some libs I use have 4/5 dependencies that I have to manually grab

**Comment by Jarrio** [comment]
might be cool if i could have a file where i can tell gsm where to grab the libs/commits from on github
